lib/state_machine/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "hyped_state_machine"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+std = []
+
+[dependencies]
+heapless = "0.8.0"
+hyped_core = { path = "../core" }
+rust-mqtt = { version = "0.3.0", default-features = false, features = ["defmt"] }
+embassy-net = { version = "0.4.0", default-features = false, features = ["defmt", "tcp", "proto-ipv4", "medium-ip"] }
```

```suggestion
embassy-net = { version = "0.4.0", default-features = false, features = ["defmt", "tcp", "proto-ipv4", "medium-ip"], git = "https://github.com/embassy-rs/embassy", rev = "1c466b81e6af6b34b1f706318cc0870a459550b7" }
```
Need to add this git pin because the version of Embassy published for Cargo is quite out of date